Gemfile.lock

```diff
@@ -630,13 +630,13 @@ GEM
       rspec (~> 3.0)
     rspec_junit_formatter (0.4.1)
       rspec-core (>= 2, < 4, != 2.12.0)
-    rubocop (0.93.1)
+    rubocop (0.92.0)
```

Comment:
Why?
![image](https://user-images.githubusercontent.com/11318903/99823419-a7f48800-2b54-11eb-90e1-323290bf9562.png)

Reply:
So it's aligned with https://github.com/3scale/porta/pull/2265/files#diff-e97904d8a0443f46ad2c8ceb5fbe5355e52f6be540378ab8ea1e98870395d7deR15

Reply:
I'm trying to figure out why Circleci throws an error locally but passes here

Reply:
Can we try going v1.x? E.g.

https://rubygems.org/gems/rubocop/versions/1.9.1
https://github.com/codeclimate/codeclimate-rubocop/branches/all?query=channel%2Frubocop-1-9